### PR TITLE
update lockbot

### PR DESCRIPTION
**Pull request by StrikerRUS**
Seems that I misunderstood the way this bot locks issues [here](https://github.com/microsoft/LightGBM/pull/2815#issuecomment-590464219).
As it locks **only closed** issues, I think that removed in this PR labels can be allowed to be locked as we keep them **open** when they are actual.

### CODE REVIEW

**Comment by guolinke** [comment]
maybe our config is wrong. In my expectation, it should close and lock the old issues..

File: .github/lock.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ daysUntilLock: 60
 skipCreatedBefore: false
 
 # Issues and pull requests with these labels will be ignored. Set to `[]` to disable
-exemptLabels: ['blocking', 'bug', 'effectiveness', 'efficiency', 'feature request', 'help wanted', 'in progress']
+exemptLabels: ['effectiveness', 'efficiency', 'feature request', 'help wanted']
 
 # Label to add before locking, such as `outdated`. Set to `false` to disable
 lockLabel: true
```
Comment (guolinke) [inline .github/lock.yml]: maybe this should set to `false`?

**Comment by StrikerRUS** [comment]
@guolinke 
> In my expectation, it should close and lock the old issues..

According to bot's description, it locks only closed issues.
> GitHub App that locks closed issues and pull requests after a period of inactivity

File: .github/lock.yml (hunk 1 above)
Comment (StrikerRUS) [reply .github/lock.yml]: As we do not have `outdated` label or any similar to this, I think you are right.

**Comment by guolinke** [comment]
@StrikerRUS yeah, but from its recent logs, it seems there are only comments, not locks, like this one:
image

**Comment by StrikerRUS** [comment]
@guolinke 
> it seems there are only comments, not locks, like this one:

I think you can comment because you are the owner of this repo. I guess ordinary users cannot comment after that.

Or maybe the bot lacks of rights to perform a loc?..

**Comment by StrikerRUS** [comment]
It looks different in XGBoost repo. We do not have the following line 

image

**Comment by guolinke** [comment]
The permission is good:
image

maybe related to `lockLabel`.

**Comment by StrikerRUS** [comment]
Perhaps. Let's merge and see 🙂
